liota/dccs/iotcc.py

```diff
@@ -488,10 +501,24 @@ def read_entity_file(self, res_uuid):
             log.error('Read file error')
         return prop_dict
 
+    def merge_prop_dict_list(self, prop_dict, prop_list):
+        # prop_dict: new property dictionary
+        # prop_list: list of dictionary items
+        if (prop_list is None):
+            return prop_dict
+        if (prop_dict is None):
+            prop_dict = {}
+        for item in prop_list:
+            for key in item:
+                if key.find("System Properties|") == -1:
```

Comment:
Why are we skipping System Properties? Can we assume they are already in sync with Airwatch?

Reply:
This "System Properties|xxx" are generated by vRops itself. If you get_properties(), you can get them. In fact, you can even see it through vRops UI.  But these are definitely not what we and AW want. So I filter them out.

Reply:
moved it into helix adapter.